zaza/openstack/charm_tests/glance_simplestreams_sync/tests.py

```diff
@@ -125,3 +125,24 @@ def _check_local_product_streams(expected_images):
         _check_local_product_streams(expected_images)
 
         logging.debug("Local product stream successful")
+
+
+class GlanceSimpleStreamsSyncWithPropertiesTest(GlanceSimpleStreamsSyncTest):
+
+    @classmethod
+    def setUpClass(cls):
+        # `setup.py:set_latest_property_config()` is required by this test and
+        # it is called by
+        # charm-glance-simplestreams-sync/tests/tests.yaml:configure phase
+        super(GlanceSimpleStreamsSyncWithPropertiesTest, cls).setUpClass()
+
+    # TODO(guimalufb) test if the latest property gets removed from old images
+    def test_200_check_image_latest_property(self):
+        """Verify that images had metadata property set"""
+        logging.debug("Checking images with latest=true property...")
+
+        filter_properties = {'filters': {'latest': 'true'}}
+        images = self.glance_client.images.list(**filter_properties)
+        self.assertTrue(len(list(images)) > 0,
+                        "'latest=true' property not found in glance images"
+                        " list")
```

Comment:
This test assumes that all the images synced will have the latest property.  If all the images have the latest property, which image is the latest synced?